src/Nethermind/Nethermind.Blockchain/FullPruning/FullPruner.cs

```diff
@@ -182,6 +182,8 @@ protected virtual void RunPruning(IPruningContext pruning, Keccak statRoot)
                     writeFlags = WriteFlags.None;
                 }
 
+                writeFlags |= WriteFlags.DisableWAL;
```

Comment:
Why not start with it and then set `LowPriority` based on `_pruningConfig.FullPruningDisableLowPriorityWrites` ?

Reply:
That would do too.